modules/deployments-replicationcontrollers.adoc

```diff
@@ -18,6 +18,13 @@ A selector is a set of labels assigned to the pods that are managed by the repli
 The replication controller does not perform auto-scaling based on load or traffic, as it does not track either. Rather, this requires its replica
 count to be adjusted by an external auto-scaler.
 
+[NOTE]
+====
+It is recommended to use a `DeploymentConfig` to create a replication controller. It is not recommended to create replication controllers directly.
```

Comment:
Suggestion to avoid the word "recommend", which the [RHSSG recommends (heh) we avoid](https://redhat-documentation.github.io/supplementary-style-guide/#recommend):

```suggestion
Use a `DeploymentConfig` to create a replication controller. Do not create replication controllers directly.
```

Reply:
Thanks, reworked it some. This guidance is one that I always want to break because I sometimes feel like the "do" and "do not" are too harsh. But I think I reworked it so I'm happy with it now